bricksrc/threshold.py

```diff
@@ -0,0 +1,1275 @@
+from namespaces import TAG, BRICK, RDFS, QUDTQK
+
+
+threshold_definitions = {
+    "Threshold": {
+        "subclasses": {
+            "Lockout_Threshold": {},
+            "Lower_Threshold": {
+                "subclasses": {
+                    "Enable_Hot_Water_System_Outside_Air_Temperature_Threshold": {
+                        "tags": [
+                            TAG.Point,
```

Comment:
All thresholds should have Parameter as a tag if Threshold is a subclass of Parameter.

Reply:
I wasn't originally planning on defining Threshold as a subclass of parameter, but see the argument for it. I am proposing we add Threshold as a top level class alongside Setpoint and Parameter